CHANGES.rst

```diff
@@ -6,10 +6,41 @@
 Summary
 +++++++
 
-- The Gammapy v0.11 release is planned for March 2019.
+- Released on March 29th, 2019
+- 11 contributors
+- 2 months of work
+- 61 pull requests (not all listed below)
 
 **What's new?**
 
+Gammapy v0.11 implements a large part of the new joint-likelihood fitting
+framework, proposed in `PIG 7`_. This includes the introduction of the
+`FluxPointsDataset`, `MapDataset` and `Datasets` classes, which now represent
```

Comment:
Use double-backticks. These single-backticks will give Sphinx warnings for unresolved references, no?
We can link to API docs from the changelog, but I usually didn't because things will move around and it's no fun if the changelog breaks. But in this case it could be useful for users to have working links.

Including links to the 3 new tutorials from the changelog is useful IMO - it's the main thing users are interested in - see the new stuff in action.